tests/resources/test_branch.py

```diff
@@ -9,8 +9,9 @@
 from citrine.exceptions import NotFound
 from citrine.resources.data_version_update import NextBranchVersionRequest, DataVersionUpdate, BranchDataUpdate
 from citrine.resources.branch import Branch, BranchCollection
```

Comment:
```suggestion
from citrine.resources.branch import Branch, BranchCollection, LATEST_VER
```
Remove hard-coded string.  Several suggestions to follow.

Reply:
This would be bad practice. It would mean if we screwed up the string in branch.LATEST_VER, the tests wouldn't catch it.

That being said, it's not the worst idea to define an independent constant in the test file.